code/game/area/Space Station 13 areas.dm

```diff
@@ -1407,27 +1407,27 @@ var/list/ghostteleportlocs = list()
 	dynamic_lighting = 0
 	valid_territory = FALSE
 
-	auxport
+/area/solar/auxport
 		name = "\improper Fore Port Solar Array"
 		icon_state = "panelsA"

```

Comment:
Right here you removed a tab and replaced it with the direct path but you didn't remove the tabs of the variables below it. Repeat for other declarations below

Reply:
Ack, that's not good. I'll follow up on this after work.